app/controllers/measurements.controller.js

```diff
@@ -0,0 +1,85 @@
+const Measurement = require('../models/measurement.model');
+
+// Create and Save a new Measurement
```

Comment:
Comments are not apprecieted until you really need them. In this case they are not needed.